src/index.scss

```diff
@@ -285,6 +285,11 @@ $collapseHoverColor: #353535;
     &:last-child {
         text-align: right !important;
     }
+
+    &--duration {
```

Comment:
Yes, we're using data attributes here to store data about the type of value that's being represented, so it's not _just_ for styling. [(more info)](https://www.sitepoint.com/how-why-use-html5-custom-data-attributes/)

With that said, having both a `--duration` class and `data-glimpse-type="duration"` is redundant for styling.